omnibus/config/software/snowflake-connector-python-py3.rb

```diff
@@ -2,28 +2,26 @@
 
 dependency "pip3"
 
-default_version "2.1.3"
+default_version "2.6.0"
 
-source :url => "https://github.com/snowflakedb/snowflake-connector-python/archive/v#{version}.tar.gz",
-       :sha256 => "855ffb93a09c3cd994dab8af7c87a46038bbba103928c5948a0edcd2500f4e1a",
+
+source :url => "https://github.com/snowflakedb/snowflake-connector-python/archive/refs/tags/v#{version}.tar.gz",
+       :sha256 => "31764cb1ee30a575d8b15aa3af1df8795d30838f",
```

Comment:
This doesn't seem to be a `sha256` patch. I get:
```suggestion
       :sha256 => "bb7af6933bdd6b8b105dac304de66fdb03e0b17378d588b5be6f1026b6ce3674",
```